setup.py

```diff
@@ -113,6 +113,7 @@ def find_rel_test():
     install_requires=[
         'setuptools >= 0.6',
         "vsc-base >= 2.4.18",
+        'daemonize >= 2.4.6',
```

Comment:
hmm, should we include this as an optional dependency instead (see `extras_require`)?